tests/tx_validation_test.go

```diff
@@ -119,7 +118,7 @@ func TestValidationPoolInsert(t *testing.T) {
 		fn   func(types.TxType, txValueMap, common.Address) (txValueMap, error)
 	}{
 		{"invalidNonce", decreaseNonce},
-		{"invalidGasLimit", decreaseGasLimit},
+		{"invalidGasLimit", decreaseGasPrice},
```

Comment:
1. Name is inconsistent with the function name.
2. Also a test case for decreaseGasPriceKip71 should be added.

Reply:
1. Right, Thansk!
2. TestValidationPoolInsertKip71 test case is what you intended? or you said this name needs to be renamed?

Reply:
@kjeom We have a function named TestValidationPoolInsertKip71, but I couldn't find the test case calling that function. So, I wanted to add the test case calling the function.

Reply:
Ahh, `TestValidationPoolInsertKip71` calls the `decreaseGasPriceKip71`.

Reply:
got it.